_includes/templates/install/ubuntu-queue-pub-sub.md

```diff
@@ -0,0 +1,42 @@
+#### Google Pub/Sub Configuration
+
+To access Pub/Sub service, you first need to create an [Google cloud account](https://cloud.google.com/).
+
+To work with Pub/Sub service you will need to create a project using [this instruction](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+Create service account credentials using [this instruction](https://cloud.google.com/pubsub/docs/quickstart-py-mac#create_service_account_credentials),
```

Comment:
If I add only publisher and subscriber roles, I get the next error: 

at org.thingsboard.server.ThingsboardServerApplication.main(ThingsboardServerApplication.java:38)
Caused by: io.grpc.StatusRuntimeException: PERMISSION_DENIED: User not authorized to perform this action.
	at io.grpc.Status.asRuntimeException(Status.java:533)
	... 24 common frames omitted

Adding the editor role, I fixed this issue.